spec/rails_helper.rb

```diff
@@ -13,8 +13,10 @@
   puts e.to_s.strip
   exit 1
 end
+
 RSpec.configure do |config|
   config.use_transactional_fixtures = true
```

Comment:
I think you should remove database cleaner gem. It does the same work as `use_transactional_fixtures` set tot true. Database cleaner is an outdated tool. It is best to use only this setting.